src/components/UserDetailsTooltip/UserDetailsTooltip.js

```diff
@@ -0,0 +1,48 @@
+import React, { useCallback } from 'react';
+import { View, Text } from 'react-native';
+import { withOnyx } from 'react-native-onyx';
+import Str from 'expensify-common/lib/str';
+import lodashGet from 'lodash/get';
+import Avatar from '../Avatar';
+import Tooltip from '../Tooltip';
+import { propTypes, defaultProps } from './userDetailsTooltipPropTypes';
+import styles from '../../styles/styles';
+import ONYXKEYS from '../../ONYXKEYS';
+
+function UserDetailsTooltip(props){
+
+    const userDetails = lodashGet(props.personalDetailsList, props.accountID, props.fallbackUserDetails);
+    const renderTooltipContent = useCallback(() => 
+         (
+            <View style={[styles.alignItemsCenter, styles.ph2]}>
+                <View style={styles.emptyAvatar}>
+                    <Avatar
+                        containerStyles={[styles.actionAvatar]}
+                        source={userDetails.avatar}
+                    />
+                </View>
+
+                {String(userDetails.displayName).trim() ? <Text style={[styles.mt1, styles.textMicroBold, styles.textReactionSenders, styles.textAlignCenter]}>{userDetails.displayName}</Text> : ''}
+
+                {String(userDetails.login).trim() ? <Text style={[styles.textMicro, styles.fontColorReactionLabel]}>{Str.removeSMSDomain(userDetails.login)}</Text> : ''}
+
+        </View>
+        )
+      , [userDetails.avatar, userDetails.displayName, userDetails.login]);
+
+    return (
+        <Tooltip renderTooltipContent={(userDetails.name || userDetails.login) ? renderTooltipContent : undefined}>
```

Comment:
We can. Changing it...

Reply:
Actually, if I do it this way it may require changes to the tooltip. A callback that returns null is still evaluated as non-null for the purpose of determining if the renderTooltipContent should be used. This causes tooltips to render with empty content. 

https://github.com/Expensify/App/blob/035783a7984e0214677b5364d9351c968887bf58/src/components/Tooltip/TooltipRenderedOnPageBody.js#LL118C5-L120C7

Reply:
I'm not sure this would be the best idea, because it changes behavior of the underlying Tooltip component, and I cannot be sure what code relies on that behavior.

Reply:
Only when accountID does not map to a user and no fallback is provided (one such example is the concierge chat)

Reply:
> I think we can just make an early `return` , but do we have any use case where `login` is missing ?

I think that would work, I'll try it now

Reply:
Done

Reply:
Yes, that was the original behavior and I thought it was outside the scope of this issue to fix that. There are a few more places that did not display the tooltip originally where it looks like it should. I think if we start digging here the scope of these changes will increase significantly.

Reply:
> Hmm, I believe the concierge has an email (login), and in my opinion, we should always show the tooltip for the concierge. However, for the chat header, the tooltip is not shown for the concierge.
> 
>  Screen.Recording.2023-06-11.at.10.31.44.PM.mov

Just fixed it, had to modify ReportUtils.getIcons to return concierge email on name for tooltip.